### ISSUE TITLE
Zelda BOTW - Assertion failed

### ISSUE THREAD

**Issue by xsellier** [decaf-emu/decaf-emu#428]
Hi,

Thank you for awesome work. I will try to keep this ticket updated.
I'm trying to launch Zelda BOTW using decaf and here is the outputs:

Commit SHA1: 232b9aa6 - Merge pull request #425 from petmac/fix-incorrect-clamp-enum
Output:
```
[info] Game path /opt/game/00050000101C9500/code/U-King.rpx
[error] Failed to parse config /home/username/.config/decaf/config.json:
[error] std::exception
[error] Try deleting config.json to automatically generate a new one with default settings.
Assertion failed:
Expression: !thread->fastMutexQueue.head
File: /opt/decaf-emu/src/libdecaf/src/modules/coreinit/coreinit_fastmutex.cpp
Line: 376

Assertion failed:
Expression: thread->state == OSThreadState::Running
File: /opt/decaf-emu/src/libdecaf/src/modules/coreinit/coreinit_thread.cpp
Line: 1180

Assertion failed:
Expression: thread->state == OSThreadState::Running
File: /opt/decaf-emu/src/libdecaf/src/modules/coreinit/coreinit_thread.cpp
Line: 1180
```

**Comment by VelocityRa** [comment]
Not sure if this is needed, current Issue for BotW is #401.

>[error] Try deleting config.json to automatically generate a new one with default settings.
Might wanna do that btw, config is in ~/.config/decaf/config.json, like it says above.

**Comment by xsellier** [comment]
Thank you @VelocityRa .
Here the log with the updated informations:

```
[info] Game path /home/binogure/PlayOnLinux's virtual drives/Cemu/drive_c/Cemu/game/00050000101C9500/code/U-King.rpx
[error] Created new config at /home/binogure/.config/decaf/config.json
Assertion failed:
Expression: !thread->fastMutexQueue.head
File: /opt/decaf-emu/src/libdecaf/src/modules/coreinit/coreinit_fastmutex.cpp
Line: 376

Assertion failed:
Expression: thread->state == OSThreadState::Running
File: /opt/decaf-emu/src/libdecaf/src/modules/coreinit/coreinit_thread.cpp
Line: 1180

Assertion failed:
Expression: thread->state == OSThreadState::Running
File: /opt/decaf-emu/src/libdecaf/src/modules/coreinit/coreinit_thread.cpp
Line: 1180
```

**Comment by achurch** [comment]
Should be fixed by the fix for #401.
